src/components/Header.tsx

```diff
@@ -43,34 +151,52 @@ export const Header: React.FC<HeaderProps> = ({ navigation }) => {
         },
       )}
     >
-      <div className="mr-6 lg:hidden">
-        <MobileNavigation navigation={navigation} />
-      </div>
       <div className="relative flex flex-grow basis-0 items-center">
         <Link href="/">
-          <a className="block w-10 overflow-hidden lg:w-auto">
-            <span className="sr-only">Home page</span>
+          <a className="block w-auto overflow-hidden lg:w-auto">
             <Logo variant={theme.isDarkTheme ? 'dark' : 'light'} />
           </a>
         </Link>
       </div>
-      {!isHomePage && (
-        <div className="-my-5 mr-6 sm:mr-8 md:mr-0">
-          <Search />
+      <div
+        className={clsx('relative ml-auto hidden items-center lg:flex', {
+          'md:flex': isHomePage,
+        })}
+      >
+        <nav className="text-sm font-semibold leading-6 text-slate-700 dark:text-slate-200">
+          <ul className="flex space-x-8">
+            <NavItems />
+          </ul>
+        </nav>
+        <div className="ml-6 flex items-center border-l border-slate-200 pl-6 dark:border-slate-800">
+          <ThemeSelector panelClassName="mt-8" />
+          <a
```

Comment:
Replace this `<a>` tag with the code below:

```html
<a
  href="https://github.com/codealoy/codealoy"
  target="_blank"
  className="group"
  rel="noopener noreferrer"
>
  <span className="sr-only">Codealoy on GitHub</span>
  <GithubIcon />
</a>
```

- This will enable open the link in a new tab
- This will update the sr-only link text
- This will use `GithubIcon` component instead of inline svg

Reply:
* add this code
* also added some extra style classes

Reply:
Mobile version is not working properly.

Reply:
done